internal/rbd/rbd_attach.go

```diff
@@ -352,29 +355,27 @@ func attachRBDImage(ctx context.Context, volOptions *rbdVolume, device string, c
 }
 
 func appendNbdDeviceTypeAndOptions(cmdArgs []string, userOptions, cookie string) []string {
-	cmdArgs = append(cmdArgs, "--device-type", accessTypeNbd)
-
 	isUnmap := CheckSliceContains(cmdArgs, "unmap")
 	if !isUnmap {
 		if !strings.Contains(userOptions, useNbdNetlink) {
-			cmdArgs = append(cmdArgs, "--options", useNbdNetlink)
+			cmdArgs = append(cmdArgs, "--"+useNbdNetlink)
```

Comment:
My main concern with this patch is that the `--options` parameter seems to be dropped. Is that intentional, and does it keep working with the currently maintained Ceph versions?